src/main/scala/org/allenai/plugins/CoreSettingsPlugin.scala

```diff
@@ -1,14 +1,36 @@
 package org.allenai.plugins
 
-import com.typesafe.sbt.SbtScalariform
-import sbt.Keys._
-import sbt._
-
+import com.typesafe.sbt.{ SbtGit, SbtScalariform }
+import com.typesafe.sbt.git.{ DefaultReadableGit, GitReadonlyInterface, ReadableGit }
+import sbt.{
+  // Implicit conversion to allow `%` on String.
+  toGroupID,
+  AutoPlugin,
+  Compile,
+  Configurations,
+  Def,
+  Defaults,
+  GlobFilter,
+  IO,
+  IntegrationTest,
+  Keys,
+  MessageOnlyException,
+  PathFinder,
+  PluginTrigger,
+  Plugins,
+  Project,
+  Setting,
+  Test,
+  ThisBuild
+}
```

Comment:
We usually just do it this way. I don't think either is great, but since it's not longer than a default terminal window (25 lines), fully-qualifying seems odd.